Makefile

```diff
@@ -185,6 +185,7 @@ $(GOBIN)/golangci-lint:
 
 lint: tools
 	$(GOBIN)/golangci-lint run -v
+	CGO_ENABLED=0 GOOS=windows GOARCH=arm64 go build -buildvcs=false $(GO_BUILD_FLAGS) -o /dev/null ./cmd/flightctl
```

Comment:
why did you add this under lint? We will now have verification that it builds during the build phase, we can remove this here.


but, you need to add support here:

https://github.com/flightctl/flightctl/blob/main/hack/build_multiarch_clis.sh

So that we also get windows binaries into the CLI artifact container.

This is what I meant, that we should use that make target/script from here instead. So you don't have to maintain it at two different places. 

That one is used from: https://github.com/flightctl/flightctl/blob/main/Containerfile.cli-artifacts#L16

Reply:
removed